draft-ietf-quic-transport.md

```diff
@@ -4422,15 +4498,16 @@ A client MUST NOT reset the packet number for any packet number space after
 processing a Retry packet; {{packet-0rtt}} contains more information on this.
 
 A server acknowledges the use of a Retry packet for a connection using the
-original_connection_id transport parameter; see
+retry_source_connection_id transport parameter; see
 {{transport-parameter-definitions}}.  If the server sends a Retry packet, it
-MUST include the Destination Connection ID field from the client's first
-Initial packet in the transport parameter.
+also subsequently includes the value of the Source Connection ID field from
+the Retry packet in its retry_source_connection_id transport parameter.
 
 If the client received and processed a Retry packet, it MUST validate that the
-original_connection_id transport parameter is present and correct; otherwise, it
-MUST validate that the transport parameter is absent.  A client MUST treat a
-failed validation as a connection error of type TRANSPORT_PARAMETER_ERROR.
+retry_source_connection_id transport parameter is present and correct;
+otherwise, it MUST validate that the transport parameter is absent. A client
+MUST treat a failed validation as a connection error of type
+TRANSPORT_PARAMETER_ERROR.
```

Comment:
This contradicts line 1568
> An endpoint MUST treat any of the following as a connection error of type PROTOCOL_VIOLATION

I have a slight preference for PROTOCOL_VIOLATION over TRANSPORT_PARAMETER_ERROR, though I don't much care as long as this section matches the other one.